packages/manager/src/features/CloudPulse/Utils/FilterConfig.ts

```diff
@@ -149,7 +159,7 @@ export const DBAAS_CONFIG: Readonly<CloudPulseServiceTypeFilterMap> = {
 };
 
 export const FILTER_CONFIG: Readonly<
-  Map<string, CloudPulseServiceTypeFilterMap>
+  Map<string | undefined, CloudPulseServiceTypeFilterMap>
```

Comment:
```suggestion
  Map<string, CloudPulseServiceTypeFilterMap>
```

keep it like this only, key should not be undefined